### PR TITLE
✨ add quiz page

### PR DESCRIPTION
**Pull request by mewiemers**
- comments will be used in later
- add quiz in components in storybook
-try get request

### CODE REVIEW

File: components/quiz/Quiz.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// const [currentQuestion, setCurrentQuestion] = useState(0);
+
+// const handleAnswerClick = (isCorrect) => {
+//   if (isCorrect) {
+//     alert("AMAZING");
+//   } else {
+//     alert("Hmm, try again!");
+//   }
+// };
+
```
Comment (phlgr) [inline components/quiz/Quiz.tsx]: You should remove commented out code before merging

File: components/quiz/Quiz.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { QuizProps } from "../../utils/type";
```
Comment (phlgr) [inline components/quiz/Quiz.tsx]: The PropTypes should be declared in the same file

**Review by phlgr — CHANGES_REQUESTED** [review]

File: components/quiz/Quiz.tsx (hunk 2 above)
Comment (mewiemers) [reply components/quiz/Quiz.tsx]: > The PropTypes should be declared in the same file

don't know what you exactly mean?!

**Review by phlgr — APPROVED** [review]
GJ 🚀

File: components/quiz/Quiz.tsx (hunk 2 above)
Comment (phlgr) [reply components/quiz/Quiz.tsx]: Please take a look at your QuizButton.tsx to see what I mean

Comment (mewiemers) [reply components/quiz/Quiz.tsx]: I looked at it but no understanding, because there are different props in these files

**Pull request merged**